playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1014,10 +1147,10 @@ worker_core_mult:
 # Stanford, for example, uses edxapp_theme_name: 'stanford'
 #
 # TODO: change variables to ALL-CAPS, since they are meant to be externally overridden
-edxapp_use_custom_theme: false
-edxapp_theme_name: ""
-edxapp_theme_source_repo: 'https://{{ COMMON_GIT_MIRROR }}/Stanford-Online/edx-theme.git'
-edxapp_theme_version: 'master'
+edxapp_use_custom_theme: true
+edxapp_theme_name: 'lagunita'
+edxapp_theme_source_repo: "git@{{ COMMON_GIT_MIRROR }}:Stanford-Online/lagunita-theme.git"
```

Comment:
If we're using the public repo, we should probably reference via HTTPS instead of SSH.